tests/versioning_test.go

```diff
@@ -1481,6 +1481,8 @@ func (s *versioningIntegSuite) TestDisableLoadUserData() {
 	// First insert some data (we'll try to read it below)
 	s.addNewDefaultBuildId(ctx, tq, v1)
 
+	s.unloadTaskQueue(ctx, tq)
```

Comment:
Maybe add a comment so the reader knows why the test unloads?

Reply:
ok